src/ElectricPokemon.java

```diff
@@ -0,0 +1,49 @@
+package src;
+
+public class ElectricPokemon extends Pokemon{
+    ElectricPokemon(String name, String foodName, String sound) {
+        super.setName(name);
+        super.setFood(foodName);
+        super.setSound(sound);
+    }
+    @Override
+    public void setName(String name) {
```

Comment:
Overriding this method doesn't add anything to the method. the only thing you're doing is calling the super. If you hadn't overridden this method, the funtionality would be the same. Remember that you're inheriting this method by extending Pokemon. So you can just say `pikachu.setName("not-pikachu")` and it will use the `setName` method from pokemon.